index.bs

```diff
@@ -7652,6 +7652,122 @@ To <dfn abstract-op>Create a new supplemental public key record</dfn>, perform t
+
+: Extension identifier
+:: `confirmation`
+
+: Operation applicability
+:: [=authentication extension|Authentication=]
+
+: Client extension input
+:: A single USVString confirmationPrompt.  This string might contain the following formatting tags: <code>&lt;b&gt;&lt;/b&gt;</code> to mark the text inbetween as bold and <code>&lt;br /&gt;</code> to force a line break.  Support of these formatting tags is "best effort".  Implementations not supporting it SHALL NOT display the tags in "verbatim".
+
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+      USVString confirmationPrompt;
+    };
+    </xmp>
+
+: Client extension processing
+:: If this extension is present, the client SHALL
+       1. use a dialog to the user that makes the user aware of the confirmation to be provided (as opposed to doing a simple sign in).
+       1. display the confirmation prompt to the user. The client SHOULD
+             indicate that the confirmation prompt originates from a specific relying party
+             (as opposed to the platform itself).
+     
+       1. use the {{CollectedClientConfirmationData}} structure containing the confirmation prompt instead of using
+             the {{CollectedClientData}} structure.
+     
+       1. pass-through the extension to the authenticator (see "client extension output" below)
+       1. pass-through the "authenticator extension output" to the caller as part of the assertion
+
+
+    <xmp class="idl">
+        dictionary CollectedClientConfirmationData : CollectedClientData {
+            required CollectedClientAdditionalConfirmationData confirmationPrompt;
+        };
+    </xmp>
+
+    The {{CollectedClientConfirmationData}} dictionary inherits from
+    {{CollectedClientData}}. It contains the following additional field:
+
+    <dl dfn-type="dict-member" dfn-for="CollectedClientConfirmationData">
+        :  <dfn>confirmationPrompt</dfn> member
+        :: The full context of the confirmation to give.
+    </dl>
+
+
+    <xmp class="idl">
+        dictionary CollectedClientAdditionalConfirmationData {
+            required USVString confirmationPrompt;
+        };
+    </xmp>
+
+    The {{CollectedClientAdditionalConfirmationData}} dictionary contains the following
+    fields:
+
+    <dl dfn-type="dict-member" dfn-for="CollectedClientAdditionalConfirmationData">
+        :  <dfn>confirmationPrompt</dfn> member
+        :: The full context of the confirmation to give.
+    </dl>
+
+: Client extension output
+:: Returns the authenticator extension output string UTF-8 decoded into a USVString.
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientOutputs {
+      USVString confirmationPrompt;
+    };
+    </xmp>
+
+: Authenticator extension input
+:: The client provides the extension input encoded as a CBOR text string (major type 3).
+
+<pre>
+CDDL:
+confirmationPromptInput = (tstr)
+</pre>
```

Comment:
The key should be the extension identifier defined above, and this should define an extension to the `$$extensionInput` socket:

```suggestion
:: The [=client extension input=] encoded as a CBOR text string (major type 3).

    ```
    $$extensionInput //= (
      confirmation: tstr,
    )
    ```
```

Reply:
Done.